financial-literacy/financial-literacy.md

```diff
@@ -84,10 +84,10 @@ The most important parts of maintaining a good credit score are:
 ### Supplemental Resources
 - [Mission Economic Development Agency](https://medasf.org/newsandpress/media-center/)
    - MEDA creates opportunity for habitually underresourced families throughout the San Francisco Bay Area. MEDA is also committed to maintaining the cultural identity and enhancing the resources of the Mission District, as we help every student achieve and every family succeed as the lead agency of the Mission Promise Neighborhood.
-- [Money Essentials](http://money.cnn.com/pf/money-essentials/)
 - [MyMoney](https://www.mymoney.gov/)
+   - This website is a product of the Congressionally chartered Federal Financial Literacy and Education Commission, which is made up of more than 20 Federal entities that are coordinating and collaborating to strengthen financial capability and increase access to financial services for all Americans. 
 - [Ally Wallet Wise](http://www.allywalletwise.com/)
-
+   - Ally Wallet Wise can help you build healthy habits when it comes to your personal finances. Ally Wallet Wise have the tools and information to help you reach your financial goals, whether you’re starting out or starting over.
```

Comment:
The link and description will look like two different points. So remove `-` that's before the description

Reply:
i have made the description as a subpoint. So shall i remove the subpoint and add it in same as that of link.

Reply:
Actually it looks like a different point when I saw the preview...

Reply:
But in my file extra - is not present. shall I repush the file?
you are talking about the last one only?

Reply:
@kundan I have checked this file even and it is fine in this file too. It looks like the subpoint of the link.